.github/workflows/ci.yml

```diff
@@ -272,6 +272,10 @@ jobs:
       if: ${{ matrix.os == 'macos-latest' }}
       run: brew install coreutils
 
+    - name: Install wget on Windows
+      if: ${{ matrix.os == 'windows-latest' }}
+      run: choco install wget
```

Comment:
That fixes the missing `wget`, but it doesn't look like that is the problem.
Can you try running with `set GIT_TRACE=1` to get detailed output about what file is causing the problem?